anemUIProcessor.py

```diff
@@ -531,7 +531,7 @@ def _process_reading_phase(self, reading):
                     temp_vz = mean(self.past_vz)
                     m = np.sqrt(pow(temp_vx, 2) + pow(temp_vy, 2) + pow(temp_vz, 2))
 
-                if len(self.past_vx) < 20:
+                if len(self.past_vx) < 10:
```

Comment:
Changed this to be consistent with the commented area above -- that is, averaging over the past 10 vx, vy, vz for phi calculations (instead of 20)